config/routes.rb

```diff
@@ -37,6 +37,9 @@
         post 'add_meal', to: 'branches#create_meal'
         get :toggle_meal_active
         get :toggle_meal_inactive
+        post :order_ready
+        post :order_picked_up
+        post :order_cancelled
```

Comment:
then you can rename them as mark_`state_name`

Reply:
Done

Reply:
Done